CANTroller2/src/TemperatureLimits.h

```diff
@@ -0,0 +1,25 @@
+#ifndef TEMPERATURELIMITS_H
```

Comment:
could shove all the Temperature stuff in a single file. I don't really mind having it spread out but Soren often prefers fewer files. He thinks we get charged by the line for code...

Reply:
I think I made this to avoid circular dependencies. They're a real pain